aten/src/ATen/native/cuda/RowwiseScaledMM.cu

```diff
@@ -264,13 +270,32 @@ void f8f8bf16_rowwise_impl(
        stride_b},
       {{{{bias.has_value() ? reinterpret_cast<DtypeBias*>(bias->data_ptr())
                            : nullptr},
-         {{reinterpret_cast<DtypeScale*>(x_scale.data_ptr())},
-          {{reinterpret_cast<DtypeScale*>(w_scale.data_ptr())}}}}},
+         {{reinterpret_cast<DtypeScale*>(w_scale.data_ptr())},
```

Comment:
These are flipped?

Reply:
Yes that's because the order of the epilogues got flipped (see just above) to workaround a CUTLASS bug in this new version